### ISSUE TITLE
"Mode" dropdown has no aria-label

### ISSUE THREAD

**Issue by schanzer** [ds26gte/code.pyret.org#81]
* The text/blocks dropdown should have a label. This is also a problem for sighted users, since an anonymous dropdown that says “text” isn’t helpful. Add a <label> to the dropdown (how about “mode”?)

**Comment by ds26gte** [comment]
Fixed in 32f4144.
